docs/gadgets/trace/tcpdrop.md

```diff
@@ -202,4 +202,12 @@ This table can be generated with:
 
 ```bash
 $ go run ./pkg/gadgets/trace/tcpdrop/tracer/dropreasongen/...
-```
\ No newline at end of file
+```
+
+### Other tools showing dropped packets
+
+The following tools can be used to show dropped packets but they are not focused on containers or Kubernetes:
+
+* [dropwatch](https://github.com/nhorman/dropwatch): interactive tool using [Netlink Devlink Trap](https://www.kernel.org/doc/html/latest/networking/devlink/devlink-trap.html) to see drops packets by the NIC.
+* [BCC's tcpdrop](https://github.com/iovisor/bcc/blob/master/tools/tcpdrop_example.txt): tool using eBPF and kprobes/tracepoints to show when a socket buffer is released by the kernel.
+* [Retis' skb-drop collector](https://github.com/retis-org/retis): tool using various collectors (eBPF, skb-drop, nftables, ovs) to show the flow of packets in the kernel.
```

Comment:
Hi @amorenoz, is this an accurate one line summary for Retis in comparison with others?

Reply:
Hi @alban , thanks for including retis!
Just a small nit: eBPF is not a collector, it's the core technology we use for all collectors :-D. Otherwise lgtm.